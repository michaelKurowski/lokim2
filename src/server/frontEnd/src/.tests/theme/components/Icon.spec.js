const React = require('react')
const Icon = require('theme/components/Icon/Icon')
const renderer = require('react-test-renderer')

describe('Icon Component', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<
            Icon />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('logo icon renders correctly', () => {
        const tree = renderer.create(<
            Icon src={"logo"} />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('sizes icon renders correctly', () => {
        const tree = renderer.create(<
            Icon height={"big"} />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
})