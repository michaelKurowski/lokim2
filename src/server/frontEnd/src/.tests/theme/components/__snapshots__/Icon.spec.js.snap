// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon Component logo icon renders correctly 1`] = `
<img
  src="icons/undefined.svg"
/>
`;

exports[`Icon Component renders correctly 1`] = `
<img
  src="icons/undefined.svg"
/>
`;

exports[`Icon Component sizes icon renders correctly 1`] = `
<img
  src="icons/undefined.svg"
/>
`;
